server/src/main/java/io/kroki/server/service/D2.java

```diff
@@ -104,6 +104,9 @@ private byte[] d2(byte[] source, JsonObject options) throws IOException, Interru
       }
       commands.add("--theme=" + themeId);
     }
+    if (options.getBoolean("sketch") == true) {
+      commands.add("--sketch");
```

Comment:
I'm not sure why the smoke test is failing but it would be great to add a new test where `?sketch` is enabled to make sure that it's working as expected.

Also, make sure that D2 v0.2.5 supports this option.

https://github.com/yuzutech/kroki/blob/main/server/ops/docker/go.mod#L5

Reply:
Really thought this one would take 3 minutes... 🙃

Where's the best place to add a test? Looks like D2 is only tested in `smoke.js`, so I can add one there but I'd need to add the ability to pass options as well.

Reply:
I think it's fine to add a test in `smoke.js`.

>  but I'd need to add the ability to pass options as well.

True... but that would be useful for other diagram librairies that support options 😄

Reply:
> Really thought this one would take 3 minutes...

Spoiler alert, it never take 3 minutes 😆

Reply:
The test failure ended up having to do with my initial changes in `D2.java`. The `sketch` feature is present in `0.2.5` but I updated to the latest version anyway (currently `0.4.1`).